docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.4'
+
+services:
+  my-python-app:
+    build:
+      context: .
+      target: main-dev
+    image: my-python-app
+    ports:
+      - "5673:5673"
+    command: ${BASE_COMMAND:- main.py}

```

Comment:
during debugging, we debug files which uploaded to the container during `docker build` process
we need to add the possibility to debug files from the working directory